alphastats/gui/utils/gpt_helper.py

```diff
@@ -153,21 +149,21 @@ def get_assistant_functions(
                 "parameters": {
                     "type": "object",
                     "properties": {
-                        "protein_id": {
+                        "gene_name": {  # this will be mapped to "protein_id" when calling the function
```

Comment:
Have you actually tested this? I'm not sure whether you also changed the corresponding function to use gene names instead of protein ids, or the function was wrong before. It should be clear in both instances if it's actually using protein ids or gene names. Generally we should always use protein ids internally, as 1. gene names can change over time, 2. not all uploaded datasets contain gene names.

Reply:
https://github.com/MannLabs/alphapeptstats/pull/355/files#r1834408990
Found this in the next pr

Reply:
yes, there's some back-and-forth with this mapping ;-)